.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: ruby
+cache: bundler
+bundler_args: "--jobs 7"
+
+rvm:
+  - 2.1.5
```

Comment:
The reason I have selected `2.1.5` is that travis does not have `2.1.6` natively installed yet. Using `2.1.6` makes Travis install Ruby everytime it is running the tests and lengthens the tests by 1 - 2 minutes.